### DISCUSSION TITLE
Is Windows 10 supported?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [copy/v86#937] [Q&A] [answered]
I see issues about Windows 8, but not many about Windows 10. Is that perhaps due to it working correctly or being deliberately unsupported rather than merely not being tried?

**Comment by copy** [comment, marked as answer]
At the very least it needs https://github.com/copy/v86/issues/845

Personally, I'm more interested in OSes that can be run on public websites, so it isn't a priority. A PR would be welcome though.

**Reply by chadbrewbaker** [reply]
Microsoft has a VM which is free for testing. Just need to Frankenstein it. https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/  If you stripped enough out of it, might get @NikolajBjorner to have it blessed as a Win11 test VM you can host. Going to need one anyway if Microsoft wants a Windows sandbox for ChatGPT in addition to the linux sandbox.
